projects/packages/assets/tests/php/patchwork.json

```diff
@@ -1,3 +1,3 @@
 {
-	"redefinable-internals": [ "filemtime", "is_readable" ]
+	"redefinable-internals": [ "filemtime", "is_readable", "str_starts_with" ]
```

Comment:
No need to mock it, just polyfill it in bootstrap.php or something like

```
if ( ! function_exists( 'str_starts_with' ) ) {
    function str_starts_with( ...
}
```

Reply:
👍 4eaadb626f7a0b6e834a5534b80138beac8d4ae1